<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App Practica</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(8, 0, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>CALCULADORA</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumero">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QDial" name="dialNumero">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="sliderNumero">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>134</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="etiquetaNum">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NUMERO:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinNum">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>251</y>
     <width>601</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="botonSeno">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SENO</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="botonFactorial">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FACTORIAL</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="botonPotencia">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>POTENCIA</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="botonPrimo">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ES PRIMO</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="botonCoseno">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>COSENO</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="botonSalir">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SALIR</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="botonRaiz">
      <property name="text">
       <string>RAIZ</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="botonTangente">
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TANGENTE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="etiquetaResultado">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>581</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialNumero</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumero</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonSalir</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumero</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderNumero</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dialNumero</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialNumero</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinNum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialNumero</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderNumero</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderNumero</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumero</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderNumero</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinNum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderNumero</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dialNumero</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
